index.html

```diff
@@ -79,7 +79,13 @@ <h2 class="d2l-heading-3">Components</h2>
 			</ul>
 		</li>
 		<li><a href="components/link/demo/link.html">d2l-link</a></li>
-		<li><a href="components/list/demo/list.html">d2l-list</a></li>
+		<li>
+			List
+			<ul>
+				<li><a href="components/list/demo/list.html">d2l-list</a></li>
+				<li><a href="components/list/demo/list-drag-n-drop.html">d2l-list (drag 'n' drop)</a></li>
```

Comment:
I think this should be left as a separate demo page, but could be persuaded otherwise